sim/SIM-PROTOCOL.md

```diff
@@ -673,9 +673,15 @@ battle will continue.
 If an invalid decision is sent (trying to switch when you're trapped by
 Mean Look or something), you will receive a message starting with:
 
-`|error|[Invalid choice]`
+`|error|[Invalid choice] MESSAGE`
 
-This will tell you to send a different decision.
+This will tell you to send a different decision. If your previous choice
+revealed additional information (For example: a move disabled by Imprison
+or a trapping effect), the error will be followed with a `|request|` command
+to base your decision off of:
+
+`|error|[Unavailable choice] MESSAGE`
+`|request|REQUEST`
```

Comment:
This should either be multiline code, or the first line needs to have two spaces after it to force a linebreak.

Reply:
Done